scripts/msmtpq/msmtpq

```diff
@@ -56,35 +56,37 @@ err() { dsp '' "$@" '' ; exit 1 ; }
 ## ======================================================================================
 ##
 ## only if necessary (in unusual circumstances - e.g. embedded systems),
-##   enter the location of the msmtp executable  (no quotes !!)
-##   e.g. ( MSMTP=/path/to/msmtp )
-##   and uncomment the test for its existence
-MSMTP=msmtp
-#[ -x "$MSMTP" ] || \
-#  log -e 1 "msmtpq : can't find the msmtp executable [ $MSMTP ]"   # if not found - complain ; quit
+##   export the location of the msmtp executable before running this script  (no quotes !!)
+##   e.g. ( export MSMTP=/path/to/msmtp )
+if [ "$MSMTP" = "" ] ; then  # If MSMTP is unset or empty...
+  MSMTP=msmtp
+elif [ ! -x "$MSMTP" ] ; then
+  log -e 1 "msmtpq : can't find the msmtp executable [ $MSMTP ]"   # if not found - complain ; quit
+fi
 ##
 ## set the queue var to the location of the msmtp queue directory
 ##   if the queue dir doesn't yet exist, create it (0700)
 ##     before using this script
 ##       e.g. ( mkdir msmtp.queue      )
 ##            ( chmod 0700 msmtp.queue )
 ##
-## the queue dir - modify this to reflect where you'd like it to be  (no quotes !!)
-Q=~/.msmtp.queue
-[ -d "$Q" ] || mkdir -m 0700 "$Q" || \
+## the queue dir - export this variable to reflect where you'd like it to be  (no quotes !!)
+Q=${Q:-~/.msmtp.queue}
```

Comment:
shouldn't we take the opportunity to rename it from `Q` to `MSMTP_QUEUE` or something slightly more descriptive? Especially if this is expected to be set *outside* of msmtp.

Reply:
You’re right, but it looks like these two commits [are already in](https://github.com/marlam/msmtp-mirror/commit/8139cf62076e79b38c441150f6295a03c9d41045) [the master branch](https://github.com/marlam/msmtp-mirror/commit/43f0e4bece87950fbde90f15d6b74b9b15c2d1dd), so that will have to be a change for another pull request.

Reply:
I just submitted #90 which renames `Q`. I’ll create another pull request that renames `LOG` later.